ga4gh/server/config.py

```diff
@@ -0,0 +1,3 @@
+class DefaultConfig(object):
```

Comment:
Please add a documentation comment at the top of the file and the standard ``__future__`` imports.